easybuild/easyconfigs/s/SimpleElastix/SimpleElastix-1.1.0-foss-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,116 @@
+easyblock = 'Bundle'
+
+name = 'SimpleElastix'
+version = '1.1.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://simpleelastix.github.io'
+description = "Multi-lingual medical image registration library."
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+toolchainopts = {'pic': True}
+
+builddependencies = [
+    ('CMake', '3.15.3'),
+    ('Bison', '3.3.2'),
+    ('Eigen', '3.3.7', '', True),
+    ('gtest', '1.10.0'),
+]
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ('HDF5', '1.10.5'),
+    ('PCRE', '8.43'),
+    ('SWIG', '4.0.1'),
+    ('VTK', '8.2.0', versionsuffix),
+    ('X11', '20190717'),
+    ('libjpeg-turbo', '2.0.3'),
+    ('LibTIFF', '4.0.10'),
+    ('Lua', '5.1.5'),
+    ('zlib', '1.2.11'),
+]
+
+default_easyblock = 'CMakeMake'
+
+local_itk_configopts = "-DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=OFF -DBUILD_TESTING=OFF "
+local_itk_configopts += "-DITK_WRAP_PYTHON=OFF -DModule_ITKReview=ON "
+local_itk_configopts += "-DITK_USE_SYSTEM_SWIG=ON -DITK_USE_SYSTEM_EIGEN=ON -DITK_USE_SYSTEM_HDF5=ON "
+local_itk_configopts += "-DITK_USE_SYSTEM_JPEG=ON -DJPEG_INCLUDE_DIR=$EBROOTLIBJPEGMINTURBO/include "
+local_itk_configopts += "-DJPEG_LIBRARY=$EBROOTLIBJPEGMINTURBO/lib64/libjpeg.%s " % SHLIB_EXT
+local_itk_configopts += "-DITK_USE_SYSTEM_PNG=ON -DPNG_PNG_INCLUDE_DIR=$EBROOTLIBPNG/include "
+local_itk_configopts += "-DPNG_LIBRARY=$EBROOTLIBPNG/lib/libpng.%s " % SHLIB_EXT
+local_itk_configopts += "-DITK_USE_SYSTEM_TIFF=ON -DTIFF_INCLUDE_DIR=$EBROOTLIBTIFF/include "
+local_itk_configopts += "-DTIFF_LIBRARY=$EBROOTLIBTIFF/lib/libtiff.%s " % SHLIB_EXT
+local_itk_configopts += "-DITK_USE_SYSTEM_ZLIB=ON -DZLIB_INCLUDE_DIR=$EBROOTZLIB/include "
+local_itk_configopts += "-DZLIB_LIBRARY=$EBROOTZLIB/lib/libz.%s " % SHLIB_EXT
+
+local_elastix_configopts = "-DCMAKE_BUILD_TYPE=Release -DELASTIX_BUILD_EXECUTABLE=OFF "
+local_elastix_configopts += "-DITK_DIR=%(builddir)s/ITK-5.0.1/easybuild_obj/"
```

Comment:
consider using local variables for the versions of ITK and elastix, to make it easier to update later (they are used in multiple places)

Reply:
fixed in e9774d4